content/posts/android/Android Tips.md

```diff
@@ -252,7 +254,8 @@ Google Fi provides [opportunistic end&#8209;to&#8209;end encryption](https://fi.
 
 This is not without its caveats:
 - Google Fi requires Play Services and the [Fi app](https://play.google.com/store/apps/details?id=com.google.android.apps.tycho&hl=en_US) to work properly. Without Play Services, all of the features mentioned above, along with visual voicemail will not work. SMS messages will have random strings added at the end of each of them.
-- On GrapheneOS, Fi VPN and end to end encrypted calls with Fi will not work. Fi VPN requires privileged integration with the operating system which GrapheneOS developers are not willing to bundle, so it will likely never work in the foreseeable future. I am not entirely sure why end to end encrypted calls with FI are not working at the moment.
+- On GrapheneOS, Fi VPN will not work. Fi VPN requires privileged integration with the operating system which GrapheneOS developers are not willing to bundle, so it will likely never work in the foreseeable future. 
+- Google Fi Wifi-calling does not work behind a VPN.
```

Comment:
Needs to be more specific. Does not work when _any_ VPN profile is enabled? Does not work when the VPN profile advertises a route for all traffic? Does not work when the OS is configured to block traffic outside of the VPN?

Reply:
Okay, so
- The Fi app gotta be in the owner profile iirc, dont even mention it
- It only breaks with the killswitch on
- if you dont disable the killswitch, you don't even need split tunneling